docs/x509/reference.rst

```diff
@@ -2319,6 +2319,77 @@ X.509 Extensions
 
         :type: int
 
+.. class:: IssuingDistributionPoint(only_contains_user_certs,\
+           only_contains_ca_certs, indirect_crl, only_contains_attribute_certs,\
+           only_some_reasons, full_name, relative_name)
```

Comment:
These should be in a different order, either the one from the RFC, or all the only "only"s togther, or something else, but right now they look basically random.